First2.0.Application/Models/HistoricoModel/HistoricoModelBase.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace First2._0.Application.Models.HistoricoModel
+{
+    public class HistoricoModelBase
```

Comment:
Classe base deve ter um abstract para o usuário não poder instanciar a mesma, a classe base sempre vai ser herdada.
```suggestion
    public abstract class HistoricoModelBase
```